android/src/main/java/com/amplitude/android/Timeline.kt

```diff
@@ -138,11 +138,9 @@ class Timeline(
         val sessionEvents = mutableListOf<BaseEvent>()
         val configuration = amplitude.configuration as Configuration
         // If any trackingSessionEvents is false (default value is true), means it is manually set
-        @Suppress("DEPRECATION")
```

Comment:
val trackingSessionEvents = AutocaptureOption.SESSIONS in configuration.autocapture

Reply:
I think I just have to remove the comment on line 140. we no longer need `trackingSessionEvents` since I'm using `AutocaptureOption.SESSIONS in configuration.autocapture` directly.

Reply:
I mean give assign AutocaptureOption.SESSIONS in configuration.autocapture to a variable.

Reply:
May I ask why we would want to assign it to a variable? Can't we use it directly?

Reply:
Just for simplifying the code. it makes the if statement too long.